active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+# Number box
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Number box is a numerics only text control with support for input formatting, validation, basic calculation, and increment stepping.
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox)
+
+## Is this the right control? 
+
+Use a **NumberBox** control to recieve and display formatted numeric values such as currency, IP addresses, and telephone numbers. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across order of operations on entered equations. 
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
```

Comment:
Will this automatically set the [InputScope](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.TextBox.InputScope) property to Number, or does that need to be set here?

Reply:
The Number InputScope should be the default, however if **AcceptsCalculation** is set to true, whatever numeral keyboard which also supports the Plus, Asterisk, Hyphen, and Slash keys will need to be visible I would have thought

Reply:
`Number` seems like a good default here.

The `Number` InputScope already includes the keys for basic arithmetic (on desktop at least) as defined within this spec. I don't think there's any need to automatically change the default InputScope based on other property settings.

Reply:
The [`FormulaNumber`](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Input.InputScopeNameValue) InputScope seems appropriate here

Reply:
`FormulaNumber` includes (on the desktop at least) lots of symbols that aren't supported by this spec and also doesn't include space, while all examples (here and in the original issue) shows spaces around the operators for clarity of reading.
![image](https://user-images.githubusercontent.com/189547/58689030-e1906c80-837d-11e9-87af-fc6a64f7ba13.png)

Reply:
[Answered here. ](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/savoyschuler/numberbox/active/NumberBox/NumberBox.md#behavioral-components) Number it is!